### ISSUE TITLE
Add Tooltip Vue component

### ISSUE THREAD

**Issue by ricardobalk** [ricardobalk/www#68]
It is quite common to use the `title` attribute on HTML elements, which will show a tooltip on almost all modern browsers. The problem with that is that it is not meant for tooltips and could give some accessibility problems (e.g. screen readers...). Therefore, there should be a custom tooltip component, made for this purpose.
